Dockerfile.template

```diff
@@ -78,7 +83,13 @@ ENV MARIADB_VERSION %%MARIADB_VERSION%%
 # (https://downloads.mariadb.org/mariadb/+releases/)
 
 RUN set -e;\
-	echo "deb https://ftp.osuosl.org/pub/mariadb/repo/$MARIADB_MAJOR/ubuntu %%SUITE%% main" > /etc/apt/sources.list.d/mariadb.list; \
+	{ \
+		if [ -n "${REPOSITORY}" ]; then \
+			echo "deb ${REPOSITORY}"; \
```

Comment:
Wouldn't you still want this to include `%%SUITE%%` (and possibly even `main`)?

Also, for this to work at all, won't it need `MARIADB_VERSION` to be an arg?  (If you've got to edit the `Dockerfile` to use it anyhow, it seems counter-intuitive to even bother with an `ARG` IMO :smile:)